src/app/@ansyn/ol/plugins/annotations/annotations.visualizer.ts

```diff
@@ -515,7 +515,7 @@ export class AnnotationsVisualizer extends EntitiesVisualizer {
 		if (entity) {
 			entity.originalEntity = merge({}, entity.originalEntity, props);
 			this.events.updateEntity.next(entity.originalEntity);
-			this.source.refresh();
+			this.source.changed();
```

Comment:
why change from refresh?

Reply:
double click on the map zooms in - we don't won't this behavior

Reply:
![image](https://user-images.githubusercontent.com/18442671/70635241-40437680-1c3c-11ea-80fc-3eefd6a2848d.png)
maps seems black when changing between active maps

Reply:
when zooming out on base map we see black tile - probably because of the deletion of:
loadTilesWhileInteracting: true, loadTilesWhileAnimating: true
we need to prevent this behavior - we should not see black tiles

Reply:
Their documentation says that 'loadTilesWhileInteracting' and 'loadTilesWhileAnimating' are no longer used and it's supposed to be automatic, but I'll try...

Reply:
I see this happening in the current version too...